2Weekly Mission/signin.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="signstyle.css" />
+    <link
+      rel="stylesheet"
+      as="style"
+      crossorigin
+      href="https://cdn.jsdelivr.net/gh/orioncactus/pretendard@v1.3.9/dist/web/static/pretendard.min.css"
+    />
+    <title>signin</title>
+  </head>
+  <body>
+    <main>
+      <article class="logintable">
+        <div class="logoNsignup">
+          <a href="/">
+            <img class="logo" src="landing/logo.svg" alt="로고" />
+          </a>
+          <div>
+            회원이 아니신가요?
+            <a class="tosignup" href="/signup.html"> 회원 가입하기</a>
+          </div>
+        </div>
+        <form>
+          <label for="email">이메일</label>
+          <input id="email" name="email" type="email" />
+          <label for="password">비밀번호 </label>
```

Comment:
불필요한 공백 들어가지 않게 확인해주세요! 전체적으로 몇몇 있네요

```suggestion
          <label for="password">비밀번호</label>
```